swan-lake/learn/keeping-ballerina-up-to-date.md

```diff
@@ -12,87 +12,91 @@ redirect_from:
+Patch releases of Ballerina distributions contain bug fixes and fixes for critical stability and security related issues. These releases are strictly time-bound and happen every two weeks. Occasionally,  you would see on-demand patch releases as well.
 
 *Example patch releases: 1.0.6, 1.1.5, 1.1.10*
 
-#### Minor Release Channel
+#### Minor Releases
 
-This channel gives you access to feature releases of Ballerina distributions. Ballerina programs that you’ve written today should continue to work on these minor releases. There will be four minor releases a year. You will get access to a minor release on the 3rd Wednesday of the 3rd month of every quarter.
+These are feature releases of Ballerina distributions. Ballerina programs that you’ve written today should continue to work on these minor releases. There will be four minor releases a year. You will get access to a minor release on the 3rd Wednesday of the 3rd month of every quarter.
 
 *Example minor releases: 1.1.0, 1.2.0, 1.3.0*
 
 #### Release Maintenance
 
-- We maintain a minor release 1.x.0 by issuing a series of patch releases 1.x.y. The maintenance of a particular minor release stops when there are two newer minor releases available.
-- In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, maintaining 1.0.0 will be stopped.
+In the 1.x stable release channel, a minor release 1.x.0 is manitained by issuing a series of patch releases as 1.x.y. The maintenance of a particular minor release stops when there are two newer minor releases available.
 
-## Keeping Ballerina Upto Date
+In other words, patch releases for 1.x.0 stop when jBallerina 1.(x+2).0 is released. E.g., when 1.2 is available, maintaining 1.0.0 will be stopped.
 
-Now that you are familiar with the terminology, let’s look at how you can keep your Ballerina distributions up to date.
+However, Swan Lake preview releases will not be maintained. Once the Swan Lake GA is released, it's a stable release. Until that, 1.2.x releases will be maintained.
 
-- The first step is to install Ballerina. Visit our [installation guide](/swan-lake/learn/installing-ballerina/) guide for details. Once the installation is complete, you would see the following directory structure inside the installation directory.
+## Installing Ballerina
+
+If you haven’t installed Ballerina yet, see [Installing Ballerina](/swan-lake/learn/installing-ballerina/) for the instructions.
+
+Once the installation is complete, you would see the directory structure below inside the installation directory.
 
 ```sh
 .
-├── bin/
-│   └── ballerina
-├── lib/
-│   └── ballerina-command-0.8.0.jar
-├── dependencies/
-│   └── jdk8u202-b08-jre/
-└── distributions/
-    ├── ballerina-version
-    ├── jballerina-1.0.5/
-    ├── jballerina-1.1.0/
-    └── jballerina-1.1.1/
+├── bin
+│   └── ballerina
+├── dependencies
+│   └── jdk8u202-b08-jre
+├── distributions
+│   ├── ballerina-slp4
+│   ├── jballerina-1.1.0-alpha
```

Comment:
I guess you have copied this from current local system. Having alpha versions in documentation is not good IMO

Reply:
Changed `jballerina-1.1.0-alpha` to `jballerina-1.1.0` as follows:
```
.
├── bin
│   └── ballerina
├── dependencies
│   └── jdk8u202-b08-jre
├── distributions
│   ├── ballerina-slp4
│   ├── jballerina-1.1.0
│   ├── jballerina-1.2.8
│   └── ballerina-version
└── lib
    └── ballerina-command-0.8.8.jar
```